### PR TITLE
Issue35 coc contrib issues

### PR DESCRIPTION
**Pull request by gdequeiroz**
Resolves Issue #35

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Before you make a substantial pull request, you should always file an issue and
+make sure someone from the team agrees that it’s a problem. If you’ve found a
+bug, create an associated issue and illustrate the bug with a minimal 
+[reprex](https://www.tidyverse.org/help/#reprex).
+
+### Pull request process
+
+*  We recommend that you create a Git branch for each pull request (PR).  
+*  Look at the Travis and AppVeyor build status before and after making changes.
```
Comment (missaugustina) [inline .github/CONTRIBUTING.md]: Are we using Appveyor? I saw the Travis badge...

File: README.Rmd [hunk 2]
```diff
@@ -71,6 +71,8 @@ Take a look at some resources:
 - [https://www.meetup.com/meetup_api/](https://www.meetup.com/meetup_api/)
 - [https://www.meetup.com/meetup_api/clients/](https://www.meetup.com/meetup_api/clients/)
 
+
+
 In order to run our tests, you will have to set the `urlname` for meetup you belong to as an environment variable using the following code:
```
Comment (missaugustina) [inline README.Rmd:76]: This is no longer true! I should update this... ;D

File: README.md [hunk 3]
```diff
@@ -1,78 +1,101 @@
...
+Once you have your key, save it as an environment variable by running
+the following:
 
 ``` r
 Sys.setenv(MEETUP_KEY = "PASTE YOUR MEETUP KEY HERE")
 ```
 
-If you don't want to save it here, you can input it in each function using the `api_key` parameter (just be sure not to send any documents with your key to GitHub 🙊).
+If you don’t want to save it here, you can input it in each function
+using the `api_key` parameter (just be sure not to send any documents
```
Comment (missaugustina) [inline README.md:43]: Note, file an issue to create a CI check for this.

**Review by missaugustina — APPROVED** [review]
This is a great start and very thorough! I added comments for some issues we need to create based on what's in here.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (gdequeiroz) [reply .github/CONTRIBUTING.md]: We are not. Good catch! I will change that.

**Comment by gdequeiroz** [comment]
Closes #35

**Pull request merged**